docs/user/Kubeconfig.md

```diff
@@ -0,0 +1,52 @@
+# Kubernetes Configuration File Types
+
+## Overview
+
+### Notes - viya4-deployment:6.6.0
```

Comment:
Are we now keeping track of versioned information in these markdown files? This is going to add quite a bit of support and update for each deployment. Again, not opposed just pointing out that over the long-term this fill will get pretty big capturing version info.

Reply:
I noticed that we used versions in existing doc already like https://github.com/sassoftware/viya4-iac-gcp/pull/169/files#diff-8a22ddc8bcdf8b5b392d4e76e637e4e6fd79cba9ebf343dddef54713b947a293
It does seem helpful to include the version here since our doc is meant to provide relevant info for current and past versions. 
In this case the version is meant to imply 6.6.0 or later (although not stated explicity) so it would not necessarily have to be updated each release, if we included that qualifier does that make it any less of an issue?

Reply:
Alright. I am good with that.